src/frontend/src/utils/iiConnection.ts

```diff
@@ -279,3 +328,40 @@ const requestFEDelegation = async (
   );
   return DelegationIdentity.fromDelegation(sessionKey, chain);
 };
+
+export const creationOptions = (
+  exclude: DeviceData[] = [],
+  authenticatorAttachment?: AuthenticatorAttachment
+): PublicKeyCredentialCreationOptions => {
+  return {
+    authenticatorSelection: {
+      userVerification: "preferred",
+      authenticatorAttachment,
+    },
+    excludeCredentials: exclude.flatMap((device) =>
+      device.credential_id.length === 0
+        ? []
+        : {
+            id: new Uint8Array(device.credential_id[0]),
+            type: "public-key",
+          }
+    ),
+    attestation: "direct",
+    challenge: Uint8Array.from("<ic0.app>", (c) => c.charCodeAt(0)),
```

Comment:
This does not matter now since we do not validate the attestation anywhere, but it may change in the future at least for the registration flow.

Reply:
I just copied what we had been using as the defaults given from "@dfinity/identity". Want me to just pass an empty array here? Or will some keys complain about that?